package.json

```diff
@@ -13,17 +13,19 @@
     "dist"
   ],
   "scripts": {
-    "test": "mocha --compilers js:./babel-register --inline-diffs --check-leaks --ui tdd --reporter dot test",
-    "build": "babel --modules commonStrict --source-maps-inline --out-dir dist src",
+    "test": "mocha --compilers js:babel-core/register --inline-diffs --check-leaks --ui tdd --reporter dot test",
+    "build": "babel --source-maps-inline --out-dir dist src",
     "prepublish": "rm -rf dist && npm update && npm run build"
   },
   "dependencies": {
-    "shift-spec": "^2015.1.3"
+    "shift-spec": "2015.2.0"
```

Comment:
From the page you linked, it is quite obvious that that would not match. See the first example. After updating `package.json`, you have to rerun `npm install`.